src/menu/menu.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Get } from '@nestjs/common';
+import { MenuService } from './menu.service';
+import { MenuItem } from './menuitem.entity';
+
+@Controller('menu')
+export class MenuController {
+    constructor(private readonly menuService: MenuService) {}
+
+    @Get()
+    GetMenu(): Promise<MenuItem[]> {
+        return this.menuService.findAll();
```

Comment:
returned item should be awaited, see: https://github.com/goldbergyoni/nodebestpractices#-212-always-await-promises-before-returning-to-avoid-a-partial-stacktrace